site/client/src/main/scala/utils/DomHandler.scala

```diff
@@ -18,64 +20,105 @@ object DomHandler {
     nodes foreach { case (n, r) ⇒ $(n).html(r) }
   }
 
+  /** Enables the compilation button of the given exercise.
+    */
+  def enableCompileButton(e: HTMLElement): IO[Unit] = io {
+    $(e).find(".compile button").prop("disabled", false)
+  }
+
+  /** Disables the compilation button of the given exercise.
+    */
+  def disableCompileButton(e: HTMLElement): IO[Unit] = io {
+    $(e).find(".compile button").prop("disabled", true)
+  }
+
+  /** Given a method name, return the node corresponds to such method.
+    */
+  def nodeByMethod(method: String): IO[Option[HTMLElement]] = for {
+    exercises ← allExercises
+  } yield exercises.find(getMethodAttr(_) == method)
+
```

Comment:
You has re-implemented a method that already existed:

[Line 113](https://github.com/47deg/scala-exercises/pull/134/files#diff-e1b555c5df32d77153fd2dda3d1b98e8R113):
```
def findExerciseByMethod(method: String): IO[Option[HTMLElement]] = for {
    exercises ← allExercises
  } yield exercises.find(methodName(_) == Option(method))
```